src/appkit.cs

```diff
@@ -11391,7 +11391,8 @@ interface NSPathControlItem
 	}
 
 	[BaseType (typeof (NSResponder))]
-	interface NSPopover : NSAccessibilityElementProtocol, NSAccessibility {
+	interface NSPopover : NSAppearanceCustomization, NSAccessibilityElementProtocol, NSAccessibility {
+		[Obsolete ("Use GetAppearance () and SetAppearance () methods instead")]
```

Comment:
Yes, btw my intro test for that is coming (:

Please use `'` for the methods and end the sentence by a dot.

`Use the 'GetAppearance' and 'SetAppearance' methods instead.`